.changelogs/llms-link-visibility.yml

```diff
@@ -0,0 +1,3 @@
+significance: patch
+type: fixed
+entry: Use `render_block_llms/navigation-link` filter instead of `render_block` for compatibility with LLMS block visibility.
```

Comment:
Hey man can you here too write some more understandable for our users? At the end this will go into the changelog which is mostly for them.
<3